Change log v46.16.0 (#1683)
forcedotcom/salesforcedx-vscode: packages/salesforcedx-vscode/CHANGELOG.md

# 46.16.0 - September 26, 2019

## Added

#### salesforcedx-vscode-core

> - Retrieve Multiple Components by Metadata Type ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))

Reviewer: ```suggestion
- Add retrieve all instances of a single metadata type for the Org Browser ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
```

Reviewer: This is how we've called it in the Iteration Plan https://github.com/forcedotcom/salesforcedx-vscode/issues/1613

Reviewer 2: Could rephrase as:
Update Org Browser to retrieve all instances of a metadata type

#### salesforcedx-sobjects-faux-generator, salesforcedx-vscode-apex

> - Select SObject category for Refresh SObject Definitions command ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))

Reviewer: ```suggestion
- Select SObject category when running `SFDX: Refresh SObject Definitions` command ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))
```

Reviewer 2: Could rephrase as:
Allow selecting SObject category (custom, standard or all) when running SFDX: Refresh SObject Definitions

#### docs

- Add retrieve multiple components documentation for [Org Browser](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-browser) ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))

- Add links at the bottom of the articles to provide feedback and log bugs ([PR #1646](https://github.com/forcedotcom/salesforcedx-vscode/pull/1646))

- Add recommended timeout setting for [Prettier](https://forcedotcom.github.io/salesforcedx-vscode/articles/getting-started/prettier) setup ([PR #1647](https://github.com/forcedotcom/salesforcedx-vscode/pull/1647))

> - Added dropdown pre-selection setting info for use in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing) ([PR #1665](https://github.com/forcedotcom/salesforcedx-vscode/pull/1665))

Reviewer 2: Add dropdown list of pre-selection settings in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing)

Author: I don't think that sounds quite right. What about 'Add pre-selection settings for Apex auto-complete dropdown list'?

#### salesforcedx-sobjects-faux-generator, salesforcedx-vscode-apex

- Allow selecting SObject category (custom, standard or all) when running `SFDX: Refresh SObject Definitions` ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))

#### docs

- Add retrieve multiple components documentation for [Org Browser](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-browser) ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))

- Add links at the bottom of the articles to provide feedback and log bugs ([PR #1646](https://github.com/forcedotcom/salesforcedx-vscode/pull/1646))

- Add recommended timeout setting for [Prettier](https://forcedotcom.github.io/salesforcedx-vscode/articles/getting-started/prettier) setup ([PR #1647](https://github.com/forcedotcom/salesforcedx-vscode/pull/1647))

- Add pre-selection settings for [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing) auto-complete dropdown list ([PR #1665](https://github.com/forcedotcom/salesforcedx-vscode/pull/1665))

> - Add [local develop](https://forcedotcom.github.io/salesforcedx-vscode/articles/lwc/localdev) documentation for lwc ([PR #1676](https://github.com/forcedotcom/salesforcedx-vscode/pull/1676)) —Contribution by [@blythesheldon](https://github.com/blythesheldon)

Reviewer: ```suggestion
- Add [local develop](https://forcedotcom.github.io/salesforcedx-vscode/articles/lwc/localdev) documentation for LWC ([PR #1676](https://github.com/forcedotcom/salesforcedx-vscode/pull/1676)) —Contribution by [@blythesheldon](https://github.com/blythesheldon)
```
